app/src/test/java/org/smartregister/sample/fp/job/AncJobCreatorTest.java

```diff
@@ -0,0 +1,66 @@
+package org.smartregister.sample.fp.job;
+
+import com.evernote.android.job.Job;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.smartregister.job.ExtendedSyncServiceJob;
+import org.smartregister.job.ImageUploadServiceJob;
+import org.smartregister.job.PullUniqueIdsServiceJob;
+import org.smartregister.job.SyncServiceJob;
+import org.smartregister.job.ValidateSyncDataServiceJob;
+import org.smartregister.sample.fp.base.BaseUnitTest;
+
+/**
+ * Created by ndegwamartin on 07/09/2018.
+ */
+public class AncJobCreatorTest extends BaseUnitTest {
+
+    private FPJobCreator jobCreator;
+
+    @Before
+    public void setUp() {
```

Comment:
There are files here that I will not review since it seems they are not supposed to be part of this PR. Could you create an issue to track cleanup of:
- Tests
- ANC specific references + naming
- Remove resources, activities & strings that are not going to be used in this project 
- Decide on the language to use Java Vs Kotlin
- Confirm that all files follow the language decision above - @ellykits can help guide on using Kotlin with the current OpenSRP libraries

it seems that you copied over most of the files from ANC, converted to kotlin using the Android Studio & made changes as per the requires.